jupyterhub_config.py

```diff
@@ -41,6 +41,7 @@
 #  Should be a subclass of Spawner.
 c.JupyterHub.spawner_class = 'jupyterhub.spawner.LocalProcessSpawner'
 
+c.JupyterHub.implicit_spawn_seconds = 1
```

Comment:
The docs say any positive value so I guess we could go with `0.01` or something `<1` to reduce lag.

Reply:
Good idea. Done. And tested.